.cirrus.yml

```diff
@@ -86,6 +86,16 @@ build_task:
 
 validate_task:
   alias: "validate"
+  depends_on:
+    - "build"
+  cargo_cache: *ro_cargo_cache
+  targets_cache: *ro_targets_cache
+  bin_cache: *ro_bin_cache
+  setup_script: *setup
+  main_script: *main
+
+verify_vendor_task:
+  alias: "verify_vendor"
```

Comment:
Uh oh, something's wrong here.  My proposed changes should have gone into `verify_vendor_task`.  This will certainly break.

Reply:
In case it helps, based on [the old commit](https://github.com/containers/netavark/commit/8c085744431b895218872cf621ea2fe4ea717a39) and a rebase, the diff should be:

```diff
diff --git a/.cirrus.yml b/.cirrus.yml
index 9bb4b76..277ee0b 100644
--- a/.cirrus.yml
+++ b/.cirrus.yml
@@ -97,21 +97,9 @@ verify_vendor_task:
   alias: "verify_vendor"
   depends_on:
     - "build"
-  # From this point forward, all cache's become read-only - meaning
-  # any changes made in this task aren't re-uploaded to the cache.
-  # This avoids some flapping between tasks, along with the upload time.
-  pkg_cache: &ro_pkg_cache # TODO: Remove when packages included in static VM images
-    <<: *pkg_cache
-    reupload_on_changes: false
-  cargo_cache: &ro_cargo_cache
-    <<: *cargo_cache
-    reupload_on_changes: false
-  targets_cache: &ro_targets_cache
-    <<: *targets_cache
-    reupload_on_changes: false
-  bin_cache: &ro_bin_cache
-    <<: *bin_cache
-    reupload_on_changes: false
+  cargo_cache: *ro_cargo_cache
+  targets_cache: *ro_targets_cache
+  bin_cache: *ro_bin_cache
   setup_script: *setup
   main_script: *main

@@ -165,6 +153,7 @@ success_task:
   depends_on:
     - "build"
     - "validate"
+    - "verify_vendor"
     - "unit"
     - "integration"
     - "meta"
```

Reply:
whoops edited wrong block. Fixed in latest commit.